src/main/java/uk/gov/ida/matchingserviceadapter/MatchingServiceAdapterModule.java

```diff
@@ -519,46 +509,14 @@ public IdaKeyStore getIdaKeyStore(MatchingServiceAdapterConfiguration configurat
 
     @Provides
     @Singleton
-    public Client getEidasMetadataJerseyClient(Environment environment, MatchingServiceAdapterConfiguration configuration) {
-        if (!configuration.isEidasEnabled()) return null;
-
-        EidasMetadataConfiguration metadataConfiguration = configuration.getEuropeanIdentity().getAggregatedMetadata();
-        return new JerseyClientBuilder(environment)
-                .using(metadataConfiguration.getJerseyClientConfiguration())
-                .build(metadataConfiguration.getJerseyClientName());
-    }
-
-    @Provides
-    @Singleton
-    public EidasTrustAnchorResolver getEidasTrustAnchorResolver(MatchingServiceAdapterConfiguration configuration, @Nullable Client client) {
-        if (!configuration.isEidasEnabled()) return null;
-
-        EidasMetadataConfiguration metadataConfiguration = configuration.getEuropeanIdentity().getAggregatedMetadata();
-        return new EidasTrustAnchorResolver(metadataConfiguration.getTrustAnchorUri(), client, metadataConfiguration.getTrustStore());
-    }
-
-    @Provides
-    @Singleton
-    public MetadataResolverRepository getEidasMetadataResolverRepository(
-            Environment environment,
-            MatchingServiceAdapterConfiguration configuration,
-            @Nullable EidasTrustAnchorResolver trustAnchorResolver,
-            @Nullable Client client
-    ) {
-        if (!configuration.isEidasEnabled()) return new DisabledMetadataResolverRepository();
-
-        EidasMetadataResolverRepository resolverRepository = new EidasMetadataResolverRepository(
-                trustAnchorResolver,
-                configuration.getEuropeanIdentity().getAggregatedMetadata(),
-                new DropwizardMetadataResolverFactory(),
-                new Timer(),
-                new MetadataSignatureTrustEngineFactory(),
-                new MetadataResolverConfigBuilder(),
-                client
-        );
-        registerMetadataRefreshTask(environment, ofNullable(resolverRepository), Collections.unmodifiableCollection(resolverRepository.getMetadataResolvers().values()), "eidas-metadata");
-        environment.healthChecks().register("TrustAnchorHealthCheck", new EidasTrustAnchorHealthCheck(resolverRepository));
-        return resolverRepository;
+    public MetadataResolverRepository getEidasMetadataResolverRepository(Environment environment) {
+        environment.healthChecks().register("TrustAnchorHealthCheck", new HealthCheck() {
```

Comment:
I debated removing this but the spirit of the PR is to do the bare minimum to deactivate those metadata and trust anchor resolvers. There are no references to `TrustAnchorHealthCheck` outside this class, but wondered if it had clickopsed into a grafana panel sometime in the past. 
Happy to remove now that I'm considering it again.

Reply:
My feeling is remove it but I don't know if it's being pinged by anything to actually make sure the MSA is alive. I don't suppose the MSA health checks we run use it?

Maybe leave it for now, it can always be pulled it a later PR.

Reply:
Thanks! the next PR will remove it